src/common/mfu_daos.c

```diff
@@ -3857,10 +3857,12 @@ static int cont_serialize_prop_str(struct hdf5_args* hdf5,
     hid_t   attr_dspace;
     hid_t   usr_attr;
 
-    if (entry == NULL || entry->dpe_str == NULL) {
-        MFU_LOG(MFU_LOG_ERR, "Property %s not found", prop_str);
-        rc = 1;
-        goto out;
+    if (entry == NULL || entry->dpe_str == NULL ) {
+        if ( !(entry != NULL && entry->dpe_str == NULL && entry->dpe_type == DAOS_PROP_CO_LABEL)) {
+            MFU_LOG(MFU_LOG_ERR, "Property %s not found", prop_str);
+            rc = 1;
+            goto out;
+        }
```

Comment:
Really, this was an oversight during original implementation. If DAOS gives us a NULL/empty string for ANY property, then we just trust that. The client code here doesn't need to enforce any business rules. So, we could simplify to something like...
```suggestion
    if (entry == NULL) {
        MFU_LOG(MFU_LOG_ERR, "Property %s not found", prop_str);
        rc = 1;
        goto out;
```

Reply:
Thank you! Just made the change you suggested.